app/src/main/java/com/example/xpvehicles/activities/AddVehicleActivity.java

```diff
@@ -146,7 +146,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == CAPTURE_IMAGE_ACTIVITY_REQUEST_CODE) {
             if (resultCode == RESULT_OK) {
                 // by this point we have the camera photo on disk
-                Bitmap takenImage = BitmapFactory.decodeFile(photoFile.getAbsolutePath());
+//                Bitmap takenImage = BitmapFactory.decodeFile(photoFile.getAbsolutePath());
```

Comment:
delete?

Reply:
Currently unsure if I will need it to fix the issue where images aren't displayed in AddVehicleActivity